### DISCUSSION TITLE
Hall of fame tw

### DISCUSSION THREAD

**Discussion by DoctorT1807** [Crownicles/Crownicles#924] [Nouvelles idées]
Avoir un hall of fame des joueurs ayant gagné le plus de top week.

**Comment by MF4db** [comment]
Je dis oui

**Reply by BastLast** [reply]
faut vous nerf

**Reply by MF4db** [reply]
fait une grève des nerfs pliz

**Comment by Procion06** [comment]
excellente idée ^^

**Reply by MF4db** [reply]
mdrrr, nan en vrai je soutiens l'idée que ce serait intéressant d'avoir un classement apart qui répertorierait les derniers gagnants (sous forme de podium) avec un classement apart des joueurs ayant le + remporté de tws
